backend/app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
+end
```

Comment:
Connected to me seeing `require(:title, :description)` in the controller, I'm assuming you want to validate that those are present?

In Rails, the model is the place where you add those kinds of validations. In this case - `validates :title, :description, presence: true`

If you need - you can also add a uniqueness validation for the `title` column. Just don't forget to add a `uniqueness index` if that's the way you want to do it

Reply:
I'm still thinking not to add `uniqueness` to title. Since there are books that can have the same title.
Prolly need to add the book's `ISBN` as way of making it unique